.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ references:
   workspace_root: &workspace_root ~/project
 
   # Combination of https://circleci.com/orbs/registry/orb/circleci/node#commands-with-cache and https://www.gatsbyjs.org/docs/build-caching/#clearing-cache
-  gatsby_cache_key: &gatsby_cache_key gatsby-cache-{{ .Branch }}-{{ checksum "yarn.lock" }}-{{ checksum "gatsby-config.js" }}-{{ checksum "gatsby-node.js" }}
+  gatsby_cache_key: &gatsby_cache_key gatsby-cache-{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-{{ checksum "yarn.lock" }}-{{ checksum "gatsby-config.js" }}-{{ checksum "gatsby-node.js" }}
```

Comment:
@jzempel @austin94 

Since @rossmoody added static Abstract images in an earlier commit, the build ran into some caching issues that require us to clear the cache since those image files don't exist anymore. For example, see this [build log](https://app.circleci.com/pipelines/github/zendeskgarden/website/426/workflows/22f8f2b0-d482-473b-97aa-a0b0ca03316d/jobs/855).

We decided on adding a `CACHE_VERSION` number to bust the cache as outlined [here](https://support.circleci.com/hc/en-us/articles/115015426888-Clear-project-cache).